packages/core/src/state-management/repository-internals.ts

```diff
@@ -288,6 +299,8 @@ export class RepositoryInternals {
     anchorStatus$: Observable<CASResponse>
   ): Subscription {
     const stopSignal = new Subject<void>()
+    this.#numPendingAnchorSubscriptions++
+    Metrics.observe(ANCHOR_POLL_COUNT, this.#numPendingAnchorSubscriptions)
```

Comment:
there's no examples of `Metrics.observe` anywhere else in the codebase, so please let me know if I did this right @gvelez17